src/styles/blocks/card/blockscard/body.scss

```diff
@@ -0,0 +1,10 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: normal;
+  font-style: normal;
+}
+
+.body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
The class name `.body` is too generic. It would be better to use a more specific class name that describes the role of the element in your layout. This is a code quality issue.